recipe/bld.bat

```diff
@@ -0,0 +1,36 @@
+if "%ARCH%"=="32" (
+    set PLATFORM=Win32
+) else (
+    set PLATFORM=x64
+)
+
+set mpfr_root=%cd%
+cd ..
+mkdir mpir\dll\%PLATFORM%\Release
+mkdir mpir\lib\%PLATFORM%\Release
+
+REM copy mpir headers
+xcopy %LIBRARY_INC%\*.h mpir\lib\%PLATFORM%\Release\ /E
+xcopy %LIBRARY_INC%\*.h mpir\dll\%PLATFORM%\Release\ /E
+
+REM copy mpir libraries
+copy %LIBRARY_LIB%\mpir.dll mpir\dll\%PLATFORM%\Release\mpir.dll
+copy %LIBRARY_LIB%\mpir.lib mpir\dll\%PLATFORM%\Release\mpir.lib
+copy %LIBRARY_LIB%\mpir_static.lib mpir\lib\%PLATFORM%\Release\mpir.lib
```

Comment:
In Windows, you need to have the `dll`s in your PATH which means `LIBRARY_BIN` should be added to path when a conda environment is added. Even if the dll is in the original place that you compile your libraries from, it won't find it unless the dll is the current directory or in the PATH. @msarahan, correct me if I am wrong